test/e2e/network/service.go

```diff
@@ -1939,7 +1939,7 @@ var _ = common.SIGDescribe("Services", func() {
 					{
 						Name:  "agnhost",
 						Image: imageutils.GetE2EImage(imageutils.Agnhost),
-						Args:  []string{"netexec", "--http-port=80", fmt.Sprintf("--delay-shutdown=%d", gracePeriod)},
+						Args:  []string{"netexec", fmt.Sprintf("--http-port=%d", servicePort), fmt.Sprintf("--delay-shutdown=%d", gracePeriodSeconds)},
```

Comment:
this is really container port, and this requires then to change 

> 				{Port: 80, Name: "http", Protocol: v1.ProtocolTCP, TargetPort: intstr.FromInt32(80)},

to Port: servicePort and TargetPort: containerPort

and the lines below that use 80 too

Reply:
I will fix all these lines to use `servicePort` and `containerPort` as well